scripts/qa_crowbarsetup.sh

```diff
@@ -2337,7 +2337,7 @@ function oncontroller_testsetup()
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
-        ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
+        testr run $tempestoptions | tee tempest.log | subunit-2to1 | tools/colorizer.py
```

Comment:
One thing we need to make sure: does calling tempest this way work in Cloud 4, Cloud 5 and Cloud 6?